lstchain/tools/lstchain_create_irf_files.py

```diff
@@ -0,0 +1,399 @@
+            # calculate theta / distance between reco and assumed source position
+            p["events"]["theta"] = calculate_theta(
+                p["events"],
+                assumed_source_az=p["events"]["true_az"],
+                assumed_source_alt=p["events"]["true_alt"],
+            )
+            self.log.debug(p["simulation_info"])
+
+        gammas = self.mc_particle["gamma"]["events"]
+
+        self.log.info(f"Using fixed G/H cut of {self.data_sel.fixed_gh_cut}")
+
+        gammas = self.data_sel.filter_cut(gammas)
+        gammas = self.data_sel.allowed_tels_filter(gammas)
+        gammas = self.data_sel.gh_cut(gammas)
+
+        # point_like = True for point like IRFs, False for Full Enclosure IRFs
+        if self.point_like:
+            gammas = self.data_sel.theta_cut(gammas)
+            gammas = self.data_sel.true_src_fov_offset_cut(gammas)
+
+        # Binning of parameters used in IRFs
+        true_energy_bins = self.data_bin.true_energy_bins()
+        reco_energy_bins = self.data_bin.reco_energy_bins()
+        migration_bins = self.data_bin.energy_migration_bins()
+        source_offset_bins = self.data_bin.source_offset_bins()
+
+        if self.point_like:
+            # Gammapy 0.18.2 needs offset bin centers for interpolation
+            # Using just 2 'edges' like [0.2,0.6] works fine for reading the IRF but,
+            # this workaround is necessary for further analysis using gammapy.
+            if len(self.data_bin.fov_offset_bins()) != 3:
+                self.log.critical("Offset binning is not appropriate for single offset")
+
+        fov_offset_bins = self.data_bin.fov_offset_bins()
+
+        if not self.only_gamma_irf:
+            background = table.vstack(
+                [
+                    self.mc_particle["proton"]["events"],
+                    self.mc_particle["electron"]["events"],
+                ]
+            )
+
+            background = self.data_sel.filter_cut(background)
+            background = self.data_sel.allowed_tels_filter(background)
+            background = self.data_sel.gh_cut(background)
+
+            background_offset_bins = self.data_bin.bkg_fov_offset_bins()
+
+        # For a fixed gh/theta cut, only a header value is added.
+        # For energy dependent cuts, a new HDU should be created
+        # GH_CUT and FOV_CUT are temporary non-standard header data
+        extra_headers = {
+            "TELESCOP": "CTA-N",
+            "INSTRUME": "LST-" + " ".join(map(str, self.data_sel.allowed_tels)),
+            "FOVALIGN": "RADEC",
+            "GH_CUT": self.data_sel.fixed_gh_cut,
+        }
+        if self.point_like:
+            self.log.info("Generating point_like IRF HDUs")
+            extra_headers["RAD_MAX"] = str(
+                u.Quantity(self.data_sel.fixed_theta_cut * u.deg)
+            )
+            extra_headers["FOV_CUT"] = str(
+                u.Quantity(self.data_sel.fixed_source_fov_offset_cut * u.deg)
```

Comment:
Unnecessary u.Quantity. Either use `number * u.deg` or `u.Quantity(number, u.deg)`, using `u.Quantity(number * u.deg)` adds an unnecessary additional operation.

Reply:
Ok. I will amend it